### PR TITLE
Add React Native scaffold

### PR DESCRIPTION
**Pull request by jribbink**
Adds React Native scaffold

______

For contributor use:

- [x] Targeted PR against `master` branch
- [x] Linked to Github issue with discussion and accepted design OR link to spec that describes this work
- [x] Code follows the [standards mentioned here](https://github.com/onflow/flow-cli/blob/master/CONTRIBUTING.md#styleguides)
- [x] Updated relevant documentation
- [x] Re-reviewed `Files changed` in the Github PR explorer
- [x] Added appropriate labels

### CODE REVIEW

File: scaffolds.json [hunk 1]
```diff
@@ -61,5 +61,12 @@
     "commit": "5c68d818e9bccf22409feb9759021e090c055e48",
     "folder": "example",
     "type": "mobile"
+  },
+  {
+    "name": "Reat Native simple example",
```
Comment (nialexsan) [inline scaffolds.json:66]: typo
```suggestion
    "name": "React Native simple example",
```

**Review by nialexsan — APPROVED** [review]

File: scaffolds.json (hunk 1 above)
Comment (jribbink) [reply scaffolds.json]: ty ❤️

File: scaffolds.json [hunk 2]
```diff
@@ -61,5 +61,12 @@
     "commit": "5c68d818e9bccf22409feb9759021e090c055e48",
     "folder": "example",
     "type": "mobile"
+  },
+  {
+    "name": "React Native simple example",
```
Comment (chasefleming) [inline scaffolds.json:66]:
```suggestion
    "name": "FCL React Native Mobile Dapp",
```

File: scaffolds.json [hunk 3]
```diff
@@ -61,5 +61,12 @@
...
+    "repo": "https://github.com/jribbink/fcl-react-native-scaffold",
+    "description": "React Native (Expo) example demonstrating FCL and Flow interactions.",
```
Comment (chasefleming) [inline scaffolds.json:68]: React Native (Expo) mobile dapp example demonstrating FCL and Flow interactions.

**Review by chasefleming — CHANGES_REQUESTED** [review]

File: scaffolds.json (hunk 3 above)
Comment (chasefleming) [inline scaffolds.json:68]:
```suggestion
    "description": "React Native (Expo) example demonstrating FCL and Flow interactions.",
```

**Review by chasefleming — CHANGES_REQUESTED** [review]

File: scaffolds.json [hunk 4]
```diff
@@ -61,5 +61,12 @@
     "commit": "5c68d818e9bccf22409feb9759021e090c055e48",
     "folder": "example",
     "type": "mobile"
+  },
+  {
+    "name": "FCL React Native Mobile Dapp",
+    "repo": "https://github.com/jribbink/fcl-react-native-scaffold",
+    "description": "React Native (Expo) example demonstrating FCL and Flow interactions.",
```
Comment (chasefleming) [inline scaffolds.json:68]:
```suggestion
    "description": "React Native (Expo) mobile dapp example demonstrating FCL and Flow interactions.",
```

**Review by chasefleming — APPROVED** [review]

**Pull request merged**